Magento_Customer/styles/modules/_register.scss

```diff
@@ -0,0 +1,12 @@
+.register {
+    max-width: 600px;
```

Comment:
please delete this, I want to avoid overwriting this value with !important in child styles